test/helper/WebDriver_test.js

```diff
@@ -198,6 +198,13 @@ describe('WebDriver', function () {
         // e.should.be.instanceOf(AssertionFailedError);
         // e.inspect().should.include("expected element h1 'Welcome to test app' to equal 'Welcome to test app!'");
       }));
+
+    it.only('should check text is not equal to empty string of element text', () => wd.amOnPage('https://codecept.discourse.group/')
```

Comment:
ONLY!

Reply:
Your linter did a lot of fixes but it bypasses this pretty important one. 
I'm using "eslint-plugin-mocha" for this kind of checks. You can try it too )

Reply:
I'm using the `npm run lint-fix` in the repo :D